packages/color-area/src/ColorArea.ts

```diff
@@ -247,10 +247,34 @@ export class ColorArea extends SpectrumElement {
                     break;
             }
         });
-        if (deltaX) {
+        if (deltaX != 0) {
             this.inputX.focus();
-        } else if (deltaY) {
+            this.inputX.dispatchEvent(
+                new Event('input', {
+                    bubbles: true,
+                    composed: true,
+                })
+            );
+            // this.inputX.dispatchEvent(
```

Comment:
I think if we `event.preventDefault();` in line 213, to make this work a little more like `ColorSlider` then we can uncomment these `change` events.

If not, we should remove them.

Also, we should join the dispatches under a single conditional after the focus logic, something like:
```
if (deltaX != 0 || deltaY != 0) {
   // dispatch here, so it's in one place
}
```

Reply:
I did check this out, and it didn't seem to work the way it was expected. I removed the change events for this reason.